monero-outreach-docs/translations/es/archive/wallet_primer_es.md

```diff
@@ -0,0 +1,103 @@
+# Primicia del monedero de papel para Monero
+
+**_“Todo lo que necesitas para almacenar Monero son herramientas primitivas. Un papel y lapiz lo harán, pero también puedes tallarlo a piedra o moldearlo en arcilla, al estilo paleolítico, ¿Cómo es esto posible?”_**  
+_13 de Julio, 2020_
+
+Resulta que el software de un monedero, la manera más común de almacenar Monero (XMR), en el sentido de la palabra, en realidad no almacena Monero. En vez de eso, los Monero existen como registros en un archivo contable público, osea, la cadena de bloques de Monero, y tu monedero carga con una contraseña especial llamada la clave pública de gasto que permite cambiar los registros. Este artículo les dará una base y una guía en cómo utilizar una copia física de la clave privada de gasto para guardar Monero llamado monedero de papel. 
+
+### _Base Técnica_
+
+Es importante tener un modelo mental de cómo funciona un monedero de papel para estar confiado y sentirse seguro al usarlo. La clave privada de gasto que mencionamos anteriormente, en su forma hexadecimal (usando del 0 al 9 y de la 'a' a la 'f'), se ve algo así:
+
+_Clave privada de gasto:_
+**09f8225e9f6dd95457a0bca31b66a599 6e6da8be8195025817c36bf490ae0903**
+
+Representa aproximadamente 256 bits de información aleatoria, contiene muchos caracteres y es tan complicado que nadie puede calcularlo. Puedes escribirla en algún lado y ocultarla, y entonces, serás dueño de los Monero que controla.
+
+Porque la clave privada de gasto sin procesar es difícil de manejar y propensa a errores, se han creado grupos de palabras con asignaciones regulares un poco más amigables llamadas frases mnemónicas. Las dos asignaciones más importantes para Monero son las siguientes:
```

Comment:
**Sugerencia:** 
Parrafo completo:
"Debido a que la clave de gasto privada en su forma original, sin procesar, es difícil de manejar y fácil cometer errores en ella, ésta se convierte en grupos de palabras regulares, llamadas frases mnemotécnicas, que son más amigables. Las dos conversiones más importantes para Monero son las siguientes:"

Quizas la palabra conversion se ajuste mejor en lugar de "asignacion"?

Reply:
Sure we can try it out, let me change it and fix the paragraph too.

Reply:
Checked, fixed and changed throughout the document.

Reply:
Taking you suggested paragraph, I got into this suggestion:

"Debido a que la clave de gasto privada en su forma original, sin procesar, es difícil de manejar y fácil cometer errores en ella, ésta se convierte en un grupo de palabras regulares, llamada frase mnemonica, siendo más amigable. Las dos conversiones más importantes para Monero son las siguientes:"